scripts/st2bootstrap-el6.sh

```diff
@@ -170,8 +170,7 @@ If you still have questions, please contact support. Alternatively, you can use
 CentOS 6 for evaluation.
 EHD
 )
-  yum list libffi-devel 1>/dev/null 2>&1 || no_libffi_devel=$?
-  if [ ! -z "$no_libffi_devel" ]; then
+  if [ -z "$(rpm -qa libffi-devel)" ]; then
```

Comment:
I get it now.

Modified version checks if package is installed, while `yum list libffi-devel` checks if package is _available_ to be installed as a dependency.

So I think the previous logic is correct, just add `sudo` if `yum list` didn't work for you.